e2e/utils/strategy/ManageStrategyDriver.ts

```diff
@@ -0,0 +1,37 @@
+import { CreateStrategyTemplate } from './../../utils/strategy/template';
+import { DebugDriver } from './../../utils/DebugDriver';
+import { navigateTo } from './../../utils/operators';
+import { MyStrategyDriver } from './../../utils/strategy/MyStrategyDriver';
+import { Page } from 'playwright-core';
+
+export class ManageStrategyDriver {
+  constructor(private page: Page) {}
+
+  async createStrategy(config: CreateStrategyTemplate) {
+    const debug = new DebugDriver(this.page);
+    await debug.createStrategy(config);
+    await navigateTo(this.page, '/');
+    const myStrategies = new MyStrategyDriver(this.page);
+    return await myStrategies.getStrategy(1);
```

Comment:
you don't need to await in the return, it has no effect.
```suggestion
    return myStrategies.getStrategy(1);
```